plugins/SegmentEditor/stylesheets/segmentation.less

```diff
@@ -320,6 +320,7 @@ div.scrollable {
     span.compareSegment,
     .starSegment {
         flex: none;
+        display: flex;
```

Comment:
ℹ️ Force children to not place them-self inside a baseline.

Otherwise, it will cause vertical scrolling:
<img width="266" height="272" alt="image" src="https://github.com/user-attachments/assets/5419aa13-1685-4d65-a420-5ddf560d89c3" />